src/main/java/org/jenkinsci/lib/configprovider/model/ConfigFile.java

```diff
@@ -33,11 +33,13 @@ of this software and associated documentation files (the "Software"), to deal
     private final String fileId;
     protected String targetLocation;
     protected boolean replaceTokens;
+    protected boolean allowEmpty;
 
-    public ConfigFile(String fileId, String targetLocation, boolean replaceTokens) {
+    public ConfigFile(String fileId, String targetLocation, boolean replaceTokens, boolean allowEmpty) {
```

Comment:
Might want to deprecate this constructor and add a new one? Allows for backward compatibility with anyone who might use this class in a Jenkinsfile or similar.

Reply:
Corrected.

Reply:
Thanks!